server/.env

```diff
@@ -38,10 +38,32 @@ GRAFANA_API_URL="https://grafana.liftedinit.tech"
 GRAFANA_USERNAME=username
 GRAFANA_PASSWORD=secretpassword
 PROMETHEUS_DATASOURCE_ID=datasourceid
+
 # JWT specific authentication. This should be changed.
 JWT_SECRET="DO NOT USE THIS VALUE. INSTEAD, CREATE A COMPLEX SECRET AND KEEP IT SAFE OUTSIDE OF THE SOURCE CODE."
 
 # Administrator. This cannot be changed once the application is running, and
 # is required.
 ADMIN_USERNAME=admin
 ADMIN_PASSWORD=admin
+
+# ====  Metrics SCHEDULER ====
+# Scheduler cron job running. If the scheduler hasn't finished work before the
+# next job, it will skip the schedule time.
+# If this and METRICS_SCHEDULER_SECONDS is missing, scheduler is disabled.
+# If both are set, this is an error.
+METRICS_SCHEDULER_CRON="*/60 * * * * *"
```

Comment:
Is that even a valid cron string? Shouldn't it be something like `*/59 * * * *`, i.e., valid values for seconds are 0-59 and there's an additional `*` in your string for some reason?

Reply:
See the SCHEDULER_CRON value from above. I cloned the setup for Talib scheduler and agree it is a different format than traditional Linux cron.  It is parsing this a 1-minute increment, but I will test with a traditional cron format.